projects/rulenode-core-config/src/lib/locale/rulenode-core-locale.constant.ts

```diff
@@ -634,7 +635,10 @@ export default function addRuleNodeCoreLocaleEnglish(translate: TranslateService
           'message-template-required': 'Message template is required',
           'use-system-slack-settings': 'Use system slack settings',
           'slack-api-token': 'Slack API token',
-          'slack-api-token-required': 'Slack API token is required'
+          'slack-api-token-required': 'Slack API token is required',
+          'fields-cant-be-equal': "Fields can't be equal",
```

Comment:
Unique key-value pair error text already present in file: 'unique-key-value-pair-error': '\'{{valText}}\' must be different from the current \'{{keyText}}\''